lib/rules/no-unescaped-entities.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @fileoverview HTML special characters should be escaped.
+ * @author Patrick Hayes
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+var DEFAULTS = ['>', '<', '"', '\'', '}'];
```

Comment:
I omitted { because it's not actually possible to include this character inside a tag - this character begins a curly-brace section and is a syntax error. I could include it here for completeness but AFAICT would have no effect.

This does raise a good point that < is also extraneous, I should remove that.

Here's an example where quotes can bite you:

```jsx
<Foo
  a="b"
  c="d">
  e="f"
```

Perhaps you get into this state by sorting props alphabetically (this is where it has happened to me). I'm happy to update the documentation to include this example as well